config.example.json

```diff
@@ -1,5 +1,56 @@
 {
     "username": "youremailaddress@gmail.com",
     "password": "yourpassword",
-    "min_value": 100
+    "min_value": 100,
```

Comment:
Hmm. I don't know if I want such verbose logging config in the same file that users will put their username and password in to. Do you think it'll confuse them? Can the logging config live in a different file?